datalake/src/main/java/com/sequenceiq/datalake/service/sdx/status/SdxStatusService.java

```diff
@@ -205,4 +239,26 @@ public String getShortStatusMessage(StackStatusV4Response statusResponse) {
         return String.format("Stack status: %s, reason: %s, cluster status: %s, reason: %s",
                 stackStatus, stackStatusReason, clusterStatus, clusterStatusReason);
     }
+
+    public ResourceEvent getResourceEventByFLowChainFactory(String flowChainFactory) {
+        switch (flowChainFactory) {
+            case "DatalakeResizeFlowEventChainFactory":
```

Comment:
I see, switch-case doesn't support the below option:
```java
case DatalakeResizeFlowEventChainFactory.class.getName():
```

Can you please use a `Map` instead of the switch-case and use the actual classes. Something like:

```java
if (flowChainFactory == null) {
   return null;
}
return Map.of(DatalakeResizeFlowEventChainFactory.class.getSimpleName(), ResourceEvent.DATALAKE_RESIZE_IN_PROGRESS, ....).getOrDefault(flowChainFactory, null):
```

Of course you cane create a constant for the above Map.